CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.2.1-SNAPSHOT] - 2022-12-02
+
+### Added
+- New properties for Password Validator:
+
+```
+## Password Validator
```

Comment:
Since this information is not private, you can also put default values into changelog